content/docs/components/serving/tfserving_new.md

```diff
@@ -6,32 +6,150 @@ weight = 51
 
 ## Serving a model
 
-We treat each deployed model as two [components](https://ksonnet.io/docs/tutorial#2-generate-and-deploy-an-app-component)
-in your APP: one tf-serving-deployment, and one tf-serving-service.
-We can think of the service as a model, and the deployment as the version of the model.
+Below is an example of a yaml manifest that deploys a mnist model for serving.
```

Comment:
Maybe add a bit of introduction such as 

To deploy a model we create 3 resources as illustrated below
  1. A deployment to deploy the model using TFServing
  1. A K8s service to create an endpoint a service
  1. An ISTIO virtual service to route traffic to the model and expose it through the ISTIO gateway

Jus tto double check we need a K8s service even though we are using ISTIO virtual services?